external-crates/move/move-compiler/tests/move_check/control_flow/infinite_loop_with_dead_exits.exp

```diff
@@ -3,10 +3,14 @@ warning[W09005]: dead or unreachable code
    │
 13 │             if (return) break;
    │                 ^^^^^^ Invalid use of a divergent expression. The code following the evaluation of this expression will be dead and should be removed.
+   │
+   = This warning can be suppressed with with '#[allow(dead_code)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
 
 warning[W09005]: dead or unreachable code
    ┌─ tests/move_check/control_flow/infinite_loop_with_dead_exits.move:13:25
    │
 13 │             if (return) break;
    │                         ^^^^^ Unreachable code. This statement (and any following statements) will not be executed.
+   │
+   = This warning can be suppressed with with '#[allow(dead_code)]' applied to the 'module' or module member ('const', 'fun', or 'struct')
```

Comment:
Double 'with' in these messages.

Also, what would you think of possibly rewording these to be a smide less verbose; something like 
```
This warning can be suppressed with '#[allow(dead_code)]' applied to the 'module', 'const', 'fun', or 'struct'
```